packages/components/src/components/drawer/drawer.stories.ts

```diff
@@ -0,0 +1,276 @@
+          }
+        ]
+      })}
+    </div>`;
+  }
+};
+
+export const Mouseless = {
+  parameters: { controls: { exclude: ['default', 'header', 'footer', 'contained', 'no-header'] } },
+  render: (args: any) => {
+    return html`<div class="mouseless" id="header-with-btn" style="width: auto; height: 95vh; position: relative;">
+      <style>
+        #header-with-btn sd-drawer::part(header) {
+          padding-left: 0;
+        }
+      </style>
+      ${generateTemplate({
+        args,
+        constants: [
+          {
+            type: 'slot',
+            name: 'default',
+            value: `<input placeholder="Default slot"></input>`
+          },
+          {
+            type: 'slot',
+            name: 'header',
+            value: `<sd-button slot='header' variant='tertiary'>Header</sd-button>`
+          },
+          {
+            type: 'slot',
+            name: 'footer',
+            value: `<sd-button slot='footer' variant='secondary'>Footer</sd-button>`
+          },
+          {
+            type: 'attribute',
+            name: 'no-header',
+            value: false
+          }
+        ]
+      })}
+    </div>`;
+  },
+  play: async ({ canvasElement }: { canvasElement?: HTMLUnknownElement }) => {
+    const el = canvasElement?.querySelector('.mouseless sd-drawer');
+    await waitUntil(() => el?.shadowRoot?.querySelector('header'));
+
+    if (el?.shadowRoot) {
+      const header = el.shadowRoot.querySelector('header');
+      if (header) {
+        await userEvent.type(header, '{space}', { pointerEventsCheck: 0 });
+      }
+    }
+  }
+};
+
+/**
+ *By default, the drawer’s panel will gain focus when opened. This allows a subsequent tab press to focus on the first tabbable element in the drawer. If you want a different element to have focus, add the autofocus attribute to it
```

Comment:
it feels not straight forward as we use "InitialFocus", "autofocus" and "contained" in this story. maybe you can find a way to write it more clear to new users.

Reply:
Second that. I think calling the story "Autofocus" would be best.

The contained thingy is neede everywhere, unfortunately to make it work in Storybook. :(